juju/names/names_windows.go

```diff
@@ -8,6 +8,7 @@ const (
 	Juju             = "juju.exe"
 	Jujuc            = "jujuc.exe"
 	Jujud            = "jujud.exe"
+	K8sAgent         = "k8sagent.exe"
```

Comment:
Not sure we need this, but I guess it can't hurt. As there may be some tests we run on windows that expect it.

Perhaps change the contstant value to "not-available" or something.